packages/plugins/rubygem-foreman_ansible_core/rubygem-foreman_ansible_core.spec

```diff
@@ -1,29 +1,34 @@
+# Generated from foreman_ansible_core-3.0.0.gem by gem2rpm -*- rpm-spec -*-
+# template: scl
 %{?scl:%scl_package rubygem-%{gem_name}}
 %{!?scl:%global pkg_name %{name}}
 
 %global gem_name foreman_ansible_core
 
 Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 2.2.1
-Release: 1%{?foremandist}%{?dist}
+Version: 3.0.0
+Release: 1%{?dist}
```

Comment:
Is the drop of foremandist intentional? I think this happened because the scl and not the plug-in template was used.

Reply:
I took what @ekohl did, originally I kept it but when he regenerated specs, it was removed. He's travelling today so I'm not sure if we can get answer from him. I'd go with merging, we can fix it if needed in separate PR.

Reply:
also note that this is in fact smart proxy dynflow core plugin, not foreman plugin, which may be the reason